inbm/dispatcher-agent/dispatcher/source/constants.py

```diff
@@ -6,7 +6,7 @@
 
 from enum import Enum, unique
 from dataclasses import dataclass, field
-from typing import Optional
+from typing import Optional, List
```

Comment:
**Suggestion:** Consider using from typing import List instead of list to maintain consistency across the codebase. [best practice]
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
from typing import Optional
```